src/profile-logic/profile-data.js

```diff
@@ -51,17 +65,6 @@ import type {
   EventDelayInfo,
   ThreadsKey,
 } from 'firefox-profiler/types';
-
-import { bisectionRight, bisectionLeft } from 'firefox-profiler/utils/bisect';
-import {
-  assertExhaustiveCheck,
-  ensureExists,
-  getFirstItemFromSet,
-} from '../utils/flow';
-
-import { timeCode } from '../utils/time-code';
-import { hashPath } from '../utils/path';
-
```

Comment:
I moved these imports to above of type imports. I was kinda weirded out when I see imports -> type imports -> more imports after :)